app/api/api_v1/routers/unfccc_ingest.py

```diff
@@ -98,7 +98,9 @@ def start_unfccc_ingest(
         ctx = cast(UNFCCCIngestContext, context)
         # We now have to populate the download_url values...
         for doc in pipeline_ingest_input:
-            doc.download_url = ctx.download_urls[doc.import_id]
+            doc.download_url = ""
```

Comment:
```
import json 
with open("sample.json", "w") as outfile:
    outfile.write(json.dumps({'name': None}))
    
 --> {"name": null}
    
    
with open("sample.json", "w") as outfile:
    outfile.write(json.dumps({'name': ''}))
```
    --> {"name": ""}
    
Are we sure we want an empty string?

Reply:
Just as this will mean in the db_state.json we may get '' or null for the download urls